src/manifest.json

```diff
@@ -11,7 +11,8 @@
     "storage"
   ],
   "host_permissions": [
-    "https://*/*"
+    "https://*/*",
```

Comment:
Hmm, you've left these two values here meanwhile the test's manifest is changed to `*://*/*`. I think both places should use the latter value.

Reply:
Hmm, I thought about changing it to `*://*/*` also outside of tests, but figured that maybe that might be overly broad, since it would also include `isolated-app:`. On the other hand, you could of course argue that anyone can easily write their own extension containing `*://*/*` - this feels similar to the filtering we had in this extension earlier for google.com cookies.
TL;DR: Don't really have a strong opinion on this, let me know what you'd prefer :)

Reply:
I based my suggestion on the documentation that claims that `*://` matches only `http` and `https`: https://developer.chrome.com/docs/extensions/mv3/match_patterns/ . Do you know whether the documentation is outdated? It doesn't mention `isolated-app` at all, neither for `*` nor as an explicit option.